docs/regions.rst

```diff
@@ -5,22 +5,37 @@ Regions Files for ``ds9`` (`~slitlessutils.modules.Region()`)
 
 * For every WFSS image:
+
 	* For every detector in the WFSS image:
+
 		* For every source:
+
 			* For every spectral order in question:
-				#. Load the PDTs for all of the :term:`direct imaging` pixels associated with the source
-				#. Decimate over wavelength
-				#. Make temporary mask with 1 for any pixel present in the PDT file(s) for the :class:`slitlessutils.sources.Source()`
-				#. Apply a *binary closing* morphological operator from `skimage.ndimage <https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.binary_closing.html>`_ with a square kernel, whose size is set in the :class:`slitlessutils.modules.Region()` object.
-				#. Find the contours for a highly-connected binary image using `scikit-image <https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.find_contours>`_ as a closed polygon.
-				#. Write the polygon into the region file as a ``ds9`` `polygon regions <https://ds9.si.edu/doc/ref/region.html>`_
+
+				#. load the PDTs for all of the :term:`direct imaging` pixels associated with the source
+				#. :term:`decimate<decimation>` over wavelength
+				#. make temporary mask with 1 for any pixel present in the decimated PDT
+				#. apply a *binary closing* morphological operator from `skimage.ndimage <https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.binary_closing.html>`_ with a square kernel, whose size is set in the :class:`slitlessutils.modules.Region()` object
+				#. find the contours for a highly-connected binary image using `scikit-image <https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.find_contours>`_ as a closed polygon
+				#. write the polygon into the region file as a ``ds9`` `polygon regions <https://ds9.si.edu/doc/ref/region.html>`_
+
 	* Write the region file to disk.
+
 * Output a separate ``ds9`` regions file for each WFSS file and detector combination.
 
-The resulting regions will have their title as the segmentation ID and the spectral order will encoded by the color of the region.  ``Slitlessutils`` assumes the ``tab20`` colormap from `matplotlib <https://matplotlib.org/stable/tutorials/colors/colormaps.html>`_, where the bold colors are for positive orders, pastel colors are for negative orders, and the zeroth order will be white.
+The resulting regions will have their title as the segmentation ID and the spectral order will encoded by the color of the region.  ``Slitlessutils`` assumes the ``tab20`` colormap from `matplotlib <https://matplotlib.org/stable/tutorials/colors/colormaps.html>`_, where the bold colors are for positive orders, pastel colors are for negative orders, and the zeroth order will be white (see :numref:`tab20` below).  The region filename will be the WFSS image filename with the detector name appended.  In :numref:`regionsexample` we show an example of the first-order traces highlighted.  
+
+
+.. _tab20:
+.. figure:: images/ordercolors.pdf
+	:align: center
+	:alt: Spectral order colors
```

Comment:
This also did not render - again, my guess is that converting it to a PNG will help.